.devcontainer/Dockerfile

```
@@ -61,12 +61,39 @@ fi
 echo "Python symlink validated: $python_link -> $(readlink "$python_link")"
 VALIDATE_PYTHON_SYMLINK
 
+# Remove the Ubuntu minimal dpkg excludes file so that subsequent package
+# installs populate /usr/share/man and /usr/share/doc normally.  Without this,
+# every apt-get install silently strips man pages and other documentation,
+# producing the "This system has been minimized" warning whenever `man` is run.
+# Also install man-db (the `man` command and database) and groff (the troff
+# rendering pipeline required to display man pages), then reinstall git and
+# git-man to restore man pages that were already stripped at image build time.
+# mandoc is added as a lightweight alternative renderer (e.g. for CI linting).
+RUN <<'INSTALL_MAN_TOOLS'
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)